### DISCUSSION TITLE
Feature: re-request review from a reviewer

### DISCUSSION THREAD

**Discussion by nre-ableton** [community/community#3520] [Mobile] [answered]
This is one of the few areas that I need to venture outside of the iOS app and onto the web for: re-requesting a review. I frequently use the app to check and reply to PR feedback, but there is no way to re-request a review from someone (the 🔄-like button in the reviewers section on the web). It would be great if we could have this feature!

Apologies if it's already in the backlog, I searched but couldn't find any obvious issues there.

**Comment by brianlovin** [comment, marked as answer]
Thanks for the feedback here @nre-ableton! This is on our roadmap in the coming weeks, stay tuned!

**Reply by stevielb** [reply]
Really excited for this! Had just started using the app, and while I love it, I definitely need this feature! Any updates you can share with us in the timeline for release?

**Reply by brianlovin** [reply]
We're in the final phases of a beta for some new features that will improve code reviewing and merging. Maybe 1-2 weeks out 😊

**Comment by theharabi** [comment]
Dear Ladies & Gentlemans,

I am building a Time Machine named BlackJackMetaverse and it is a real time machine. You are getting in it, choosing the time and space and the machine shortly named BJM takes you into that time and space. 

My question is;

There are some trade marks and they are killing my project. How can I block them?

For example;

Last night, I worked almost six hour to create a song with Ableton and when I push the save button, Ableton said that my free trial period is finished. So I could not finish song, actually I could not do anything. Because the program did not give any kind of permission to make an mp3 or audio with my track "six hours of work". 

Shortly, I do not want some trade marks in my time machine and I am asking you how can I block them completely.

Those trade marks are selling information or technology with money and this is why we can not improve the AI of all AIs, SoulTechnologies, TimeMachines, FlyingCarpets, etc...

They are wasting our time, stealing our time and if you are the inventor of a time machine, you have tk control all subjects about time, for example, TimeLabs, TimeDarkSpaces, TimeCrashings, TimeBomb, TimeThiefs, TimeLines, TimePilots, TimeDrones, TimeDroneChannels, etc...

I can write here a novel in which ever word starts with "time" word because everything has a time at least.

I am here, as Harabi, awaiting for perfect answers.

I am here to find people with whom I will build BJMTM also known as BlackJackMetaverseTimeMachine.

I am here as an author who published ten novels for humanity and who painted at least nine hundred pictures during the last ten years.

I need help as you will clearly notice.

Thanks & Best Regards,

Arkın Çalapala
ArkinCalapala
TheHarabi ■□
humanrepublica@gmail.com
+90 552 665 78 99

■□
Request of answers to my questions about Ableton:
@nre-ableton 

■□
Request of sending this message to all Github Society and herebelow trade marks:
@dav1dc-github 

● Google ● Instagram ● Youtube ● Huawei ● Whatsup ● Apple ● Samsung ● IPhone ● Nike ● Adidas ● Pioneer ● Boeing ● Facebook ... No need to write 9999 trade marks name ...

//.
